core/include/cubos/core/ecs/world.hpp

```diff
@@ -133,14 +135,19 @@ namespace cubos::core::ecs
 
         auto entity = this->entityManager.create(mask);
         ([&](auto& component) { this->componentManager.add(entity.index, component); }(components), ...);
+
+#if CUBOS_LOG_LEVEL <= CUBOS_LOG_LEVEL_DEBUG
```

Comment:
Intended, it is still controlled at runtime through `CUBOS_DEBUG`.
The only issue is that there's the overhead of creating the strings even if we don't get to use them, but this only happens if we're compiling with debug information, so it should be okay.
What do you think?

Reply:
Ah I see what you mean. I don't think it's much of an issue since this is only compiled for a debug build, which is expected to be unoptimized and slower for the sake of debugging purposes. This overhead won't happen on release builds which is what developers should do for final products anyways.